src/custom/lib/components/Widget.tsx

```diff
@@ -0,0 +1,145 @@
+import { Provider as EthersProvider } from '@ethersproject/abstract-provider'
```

Comment:
Can you rename this file to WidgetMod and create the index file and so on?
Will be easier when we have to update it :)